__tests__/endpoints.test.js

```diff
@@ -359,4 +359,31 @@ describe("PATCH /api/articles/:article_id", () => {
             expect(body.msg).toBe("Bad Request");
         });
     });
+})
+
+describe("DELETE /api/comments/:comment_id", () => {
+    test("204: deletes the comment with the given comment id", () => {
+        return request(app)
+        .delete("/api/comments/2")
+        .expect(204)
+        .then(({body}) => {
+            expect(body).toEqual({})
+        })
```

Comment:
When we use the status code of 204 we cannot receive anything in the body so it will always be an empty object, which means you dont need to have an expect  or even a .then at all for this endpoint.